paddle/fluid/operators/recv_vars_op.cc

```diff
@@ -26,11 +26,11 @@ limitations under the License. */
 namespace paddle {
 namespace operators {
 
-class RecvOp : public framework::OperatorBase {
+class RecvVarsOp : public framework::OperatorBase {
  public:
-  RecvOp(const std::string& type, const framework::VariableNameMap& inputs,
-         const framework::VariableNameMap& outputs,
-         const framework::AttributeMap& attrs)
+  RecvVarsOp(const std::string& type, const framework::VariableNameMap& inputs,
```

Comment:
We may call them send, recv op is OK now.

Reply:
Done. Thanks!